android/res/xml/automotive_app_desc.xml

```diff
@@ -0,0 +1,3 @@
+<automotiveApp>
+    <uses name="media" />
```

Comment:
Why "media"? The documentation says it should be "template".

Reply:
Well, we can use navigation here to make use of the navigation template provided by google

Reply:
Where is "media" documented?

Reply:
I have fixed it